src/components/pushClient/Banner.jsx

```diff
@@ -23,19 +23,21 @@ class BannerClient extends Component {
     mustShow: false
   }
 
-  async componentWillMount() {
+  componentWillMount() {
+    this.willUnmount = false
     if (Config.promoteDesktop.isActivated !== true) return
```

Comment:
I think this line should be in the `componentDidMount`?

Reply:
Hmmm, not really since the render() has already been called in the DidMount. i'll move this line to the render method itself and remove the need of willmount ;)